flake-module.nix

```diff
@@ -203,13 +224,25 @@ in
       in
       {
         packages =
-          lib.mapAttrs
-            (_: project: project.package)
-            projects;
-        apps =
-          lib.mapAttrs
-            (_: project: project.app)
-            projects;
```

Comment:
`apps` were removed. It was not an essential feature afaic, and it was incomplete. It should have filtered for packages with executable, and add an app for each executable. This is non-trivial and can be added later.

Reply:
Closing in favor of #36